sml_builder/static/js/cookies_function.js

```diff
@@ -0,0 +1,189 @@
+const DEFAULT_COOKIE_CONSENT = {
+    essential: true,
+    settings: false,
+    usage: false,
```

Comment:
I think the default setting for the usage stuff be on. We won't get any tracking of users if the default is off.

Reply:
Done :)